### PR TITLE
Adding Argo Workflows documentation

### PR DESCRIPTION
**Pull request by hectormachin**
Documentation changes can be previewed at: 
* https://github.com/Element84/filmdrop-k8s-helm-charts/tree/hm/argo-documentation/charts/swoop-bundle#running-an-argo-workflow
* https://github.com/Element84/filmdrop-k8s-helm-charts/tree/hm/argo-documentation/charts/swoop-caboose#running-an-argo-workflow

### CODE REVIEW

File: charts/swoop-bundle/README.md [hunk 1]
```diff
@@ -43,14 +43,14 @@ To install SWOOP components:
 Once the chart has been deployed, you should see at least 3 deployments: postgres, minio and swoop-api.
 <br></br>
 <p align="center">
-  <img src="../../images/swoop-api-deployment-services.png" alt="SWOOP Deployment" width="1776">
+  <img src="../.../../images/swoop-api-deployment-services.png" alt="SWOOP Deployment" width="1776">
```
Comment (geomatician) [inline charts/swoop-bundle/README.md:46]: is this path correct? I'm not seeing the image load on the readme

File: charts/swoop-bundle/README.md [hunk 2]
```diff
@@ -43,14 +43,14 @@ To install SWOOP components:
...
 </p>
 <br></br>
 
 In order to start using the services used by this helm chart, you will need to port-forward `postgres` onto localhost port `5432`, port-forward `minio` onto localhost ports `9000` & `9001` and port-forward `swoop-api` onto localhost port `8000`.
 <br></br>
 <p align="center">
-  <img src="../../images/swoop-api-port-forwarding.png" alt="Port forwarding SWOOP" width="1776">
+  <img src="../.../../images/swoop-api-port-forwarding.png" alt="Port forwarding SWOOP" width="1776">
```
Comment (geomatician) [inline charts/swoop-bundle/README.md:53]: is this path correct? I'm not seeing the image load on the readme

Comment (hectormachin) [reply charts/swoop-bundle/README.md]: fixed

File: charts/swoop-bundle/README.md (hunk 1 above)
Comment (hectormachin) [reply charts/swoop-bundle/README.md]: fixed

File: charts/swoop-caboose/README.md [hunk 3]
```diff
@@ -132,6 +132,160 @@ time="2023-07-28T17:45:34Z" level=info msg="index config" indexWorkflowSemaphore
...
+
+Access permission for `swoopminio/payloadtest` is set to `public`
+```
+```
+$ mc cp ./payload_workflow.json swoopminio/payloadtest/
+
+...opy-assets-stac-task/payload_workflow.json: 4.58 KiB / 4.58 KiB ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 127.61 KiB/s 0s
+```
+
+Now modify the `secrets.yaml` and replace the `<REPLACE_WITH_BASE64_AWS_ACCESS_KEY>`, `<REPLACE_WITH_BASE64_AWS_SECRET_ACCESS_KEY>` and `<REPLACE_WITH_BASE64_AWS_REGION>` with the base64 encoded version of the AWS_ACCESS_KEY, AWS_SECRET_ACCESS_KEY and AWS_REGION of the iam user and bucket created on the [Pre-requisites section](#pre-requisites-to-running-argo-workflows-example).
```
Comment (geomatician) [inline charts/swoop-caboose/README.md:249]: also fix this link

Comment (hectormachin) [reply charts/swoop-caboose/README.md:249]: @geomatician this link is working for me, check: https://github.com/Element84/filmdrop-k8s-helm-charts/tree/hm/argo-documentation/charts/swoop-caboose#pre-requisites-to-running-argo-workflows-example

File: charts/swoop-caboose/README.md [hunk 4]
```diff
@@ -132,6 +132,160 @@ time="2023-07-28T17:45:34Z" level=info msg="index config" indexWorkflowSemaphore
...
+<br></br>
+
+And you should be able to see your asset S3 bucket populated with a tumbnail image:
+```
+aws s3 ls s3://copy-assets-stac-task-bucket/data/naip/tx_m_2609719_se_14_060_20201217/
+
+2023-08-17 10:00:08       9776 thumbnail.jpg
+```
+<br></br>
+
```
Comment (geomatician) [inline charts/swoop-caboose/README.md:277]: should we also add a statement here on how to run the `workflow_copyassets_with_template.yaml `workflow?

`kubectl apply -f ./workflow-template.yaml
`
`argo submit -n default --watch ./workflow_copyassets_with_template.yaml `

File: charts/swoop-caboose/README.md [hunk 5]
```diff
@@ -132,6 +132,160 @@ time="2023-07-28T17:45:34Z" level=info msg="index config" indexWorkflowSemaphore
...
+$ mc cp ./payload_workflow.json swoopminio/payloadtest/
+
+...opy-assets-stac-task/payload_workflow.json: 4.58 KiB / 4.58 KiB ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 127.61 KiB/s 0s
+```
+
+Now modify the `secrets.yaml` and replace the `<REPLACE_WITH_BASE64_AWS_ACCESS_KEY>`, `<REPLACE_WITH_BASE64_AWS_SECRET_ACCESS_KEY>` and `<REPLACE_WITH_BASE64_AWS_REGION>` with the base64 encoded version of the AWS_ACCESS_KEY, AWS_SECRET_ACCESS_KEY and AWS_REGION of the iam user and bucket created on the [Pre-requisites section](#pre-requisites-to-running-argo-workflows-example).
+
+Create the kubernetes secret by executing:
+```
+kubectl apply -n default -f ./secret.yaml
```
Comment (geomatician) [inline charts/swoop-caboose/README.md:253]: should this secret be called `copy-stac-asset-s3-secret` instead as it contains credentials for S3 and not MinIO?

File: charts/swoop-caboose/README.md (hunk 4 above)
Comment (hectormachin) [reply charts/swoop-caboose/README.md:289]: Added.

File: charts/swoop-caboose/README.md (hunk 5 above)
Comment (hectormachin) [reply charts/swoop-caboose/README.md:253]: Fixed.

**Review by geomatician — APPROVED** [review]

**Pull request merged**